.github/workflows/ci.yml

```diff
@@ -247,16 +246,10 @@ jobs:
         uses: actions/setup-node@v2
         with:
           cache: npm
-          node-version-file: '.node-version'
+          node-version: 18
```

Comment:
I assume we need to pin this because we are still using 17 on v16.x.x and something is broken there? Maybe we could add a comment to eventually change that...

Reply:
Next does not work with anything but v18 or higher

Reply:
```suggestion
          # TODO: switch back in v17
          # node-version-file: '.node-version'
          node-version: 18
```